src/lib/autocomplete/autocomplete.md

```diff
@@ -69,9 +68,9 @@ filter that doesn't limit matches to the beginning of the string.
 ### Setting separate control and display values
 
 If you want the option's control value (what is saved in the form) to be different than the option's
-display value (what is displayed in the actual text field), you'll need to set the `displayWith`
+display value (what is displayed in the text field), you'll need to set the `displayWith`
 property on your autocomplete element. A common use case for this might be if you want to save your
-data as an object, but display just one of the option's string properties.
+data as an object but display just one of the option's string properties in the text field.
```

Comment:
I think the comma here is necessary. Sounds like a run-on otherwise.

Reply:
I disagree. The comma indicates the remainder will be an independent clause, but it still lacks a subject.  How about:

> A common use case for this might be if you want to save your data as an object, but you want to display just one of the option's string properties in the text field.

Reply:
Sounds like a good compromise 👍